main.go

```diff
@@ -517,14 +517,21 @@ func (c *controller) populate(hashrings []receive.HashringConfig, statefulsets m
 	for i, h := range hashrings {
 		if sts, exists := statefulsets[h.Hashring]; exists {
 			var endpoints []string
+
 			for i := 0; i < int(*sts.Spec.Replicas); i++ {
+				// if cluster Domain is empty string we don't want dot after svc
+				clusterDomain := ""
+				if len(c.options.clusterDomain) != 0 {
+					clusterDomain = fmt.Sprintf(".%s", c.options.clusterDomain)
+				}
```

Comment:
```suggestion
				clusterDomain := ""
				if c.options.clusterDomain != "" {
					clusterDomain = fmt.Sprintf(".%s", c.options.clusterDomain)
				}
```

(Does not matter much, just pattern 🤗 )

Reply:
Changed :).